pkg/install.go

```diff
@@ -284,6 +277,12 @@ func InstallGoChaosEngine(testsDetails *types.TestDetails, chaosEngine *types.Ch
 		chaosEngine.Spec.Experiments[0].Spec.Components.Env = append(val, slice)
 	}
 
+	if testsDetails.LibImage == "" {
+		testsDetails.LibImage = testsDetails.GoExperimentImage
```

Comment:
We can't update every lib image to go-runner. Some lib image has `ubuntu:16:04` like kubelet svc kill.

Reply:
Yes, it will override only if LIB_IMAGE is not passed explicitly